pints/plot/_surface.py

```diff
@@ -61,14 +67,15 @@ def surface(points, values, boundaries=None, markers='+', figsize=None):
 
     # Check boundaries
     if boundaries is None:
-        xmin = 0.80 * np.min(x)
-        xmax = 1.02 * np.max(x)
-        ymin = 0.80 * np.min(y)
-        ymax = 1.02 * np.max(y)
+        xmin, xmax = np.min(x), np.max(x)
+        xmin -= 0.1 * (xmax - xmin)
+        xmax += 0.1 * (xmax - xmin)
```

Comment:
This will be the shifted xmin... I would prefer `(xmax - xmin)` being a fixed value (much easier to interprete)...

Reply:
Thanks!